confidence/io.py

```diff
@@ -47,14 +46,12 @@ def read_xdg_config_home(name: str, extension: str) -> Configuration:
     :returns: a `Configuration` instance, possibly `NotConfigured`
     """
     # find optional value of ${XDG_CONFIG_HOME}
-    config_home = environ.get('XDG_CONFIG_HOME')
-    if not config_home:
-        # XDG spec: "If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used."
-        # see https://specifications.freedesktop.org/basedir-spec/latest/ar01s03.html
-        config_home = path.expanduser('~/.config')
+    # XDG spec: "If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used."
+    # see https://specifications.freedesktop.org/basedir-spec/latest/ar01s03.html
+    config_home = Path(environ.get('XDG_CONFIG_HOME') or '~/.config').expanduser()
```

Comment:
Functionality is now changed, `expanduser()` is called on the value in `$XDG_CONFIG_HOME`, let's maybe not include that change.